src/main/cli/cli.c

```diff
@@ -4839,7 +4839,7 @@ static void cliTasks(const char *cmdName, char *cmdline)
 
 #ifndef MINIMAL_CLI
     if (systemConfig()->task_statistics) {
-        cliPrintLine("Task list             rate/hz  max/us  avg/us maxload avgload  total/ms");
+        cliPrintLine("Task list             rate/hz  max/us  avg/us maxload avgload  total/ms   late    run   reqd    max");
```

Comment:
Can we add time units to reqd and max?

Reply:
max and max/us are duplicated I think

Reply:
Done.

Reply:
3rd and the last columns are probably same. Same numbers. I realized it just after sending the message. Sorry.